mlflow/__init__.py

```diff
@@ -1,22 +1,43 @@
+"""Provides the MLflow fluent API, allowing management of an active MLflow run.
```

Comment:
Micro style nit: I would put the `"""` on its own line in this case

Reply:
I was following [Google](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) & [PEP8's](https://www.python.org/dev/peps/pep-0008/#id33) style here...

Reply:
(I don't mind changing, but I just wanted something to point to for future people -- let me know if you have a style guide in mind)

Reply:
I think PEP8 allows you to have the quotes on their own line in this case.

Reply:
Great, switched all the module comments to this form.